soc/intel/intel_adsp/ace/mmu_ace30.c

```diff
@@ -141,3 +145,19 @@ const struct xtensa_mmu_range xtensa_soc_mmu_ranges[] = {
 };
 
 int xtensa_soc_mmu_ranges_num = ARRAY_SIZE(xtensa_soc_mmu_ranges);
+
+void arch_xtensa_mmu_post_init(bool is_core0)
+{
+	ARG_UNUSED(is_core0);
+
+	/*
+	 * Mapping the Host IPC MMIO registers into TLB way 9.
+	 * The registers are R/W and ring 0 only.
```

Comment:
I'm not familiar with the TLB operation on ACE, but (1) why way 9? Is it the highest number, the least likely to be used? (2) is this function "only mapping" a TLB entry or actually permanently "pinning" it as the description is saying? Wouldn't it be sufficient to do the pinning when powering down?